src/App.js

```diff
@@ -24,45 +24,89 @@ function App() {
 		user ? setAuthStatus(true) : setAuthStatus(false);
 	});
 
-	return (
-		<Router>
-			<>
-				<MobileLogo alt="mobile logo" src={mobileLogo} />
-			</>
-			<Switch>
-				<Route path="/" exact>
-					{authStatus ? <Redirect to="/home" /> : <LandingPage />}
-				</Route>
-
-				<Route path="/home" exact>
-					{authStatus ? <UserHome /> : <Redirect to="/login" />}
-				</Route>
-
-				<Route path="/add-medication">
-					<AddMedication />
-				</Route>
-
-				<Route path="/signup" exact>
-					{authStatus ? <Redirect to="/" /> : <SignUp />}
-				</Route>
-
-				<Route path="/login">
-					{authStatus ? <Redirect to="/" /> : <Login />}
-				</Route>
-
-				<Route path="/medication-added">
-					<MedicationAdded />
-				</Route>
-				<Route path="/search">
-					<SearchNHS />
-				</Route>
-
-				<Route>
-					<PageNotFound />
-				</Route>
-			</Switch>
-		</Router>
-	);
+	if (authStatus) {
+		return (
+			<Router>
+				<>
+					<MobileLogo alt="mobile logo" src={mobileLogo} />
+				</>
+				<Switch>
+					<Route path="/" exact>
+						<Redirect to="/home" />
+					</Route>
+
+					<Route path="/home">
+						<UserHome />
+					</Route>
+
+					<Route path="/add-medication">
+						<AddMedication />
+					</Route>
+
+					<Route path="/signup">
+						<Redirect to="/home" />
+					</Route>
+
+					<Route path="/login">
+						<Redirect to="/home" />
+					</Route>
+
+					<Route path="/medication-added">
+						<MedicationAdded />
+					</Route>
+
+					<Route path="/search">
+						<SearchNHS />
+					</Route>
+
+					<Route>
+						<PageNotFound />
+					</Route>
+				</Switch>
+			</Router>
+		);
+	} else {
```

Comment:
don't neccessarily need to explicitly write the else statement, as the return in the first if statement will stop the rest of the code from running anyways (if it does run)